### DISCUSSION TITLE
Is it still possible to register as a copilot without paying?

### DISCUSSION THREAD

**Discussion by putrifatrilyam** [community/community#19949] [Copilot Conversations]
I signed up for github copilot for free, but my friend when he wanted to register it couldn't be free anymore

**Comment by kevinalfito69** [comment]
You can read more about this at : https://github.com/pricing#i-work-on-open-source-projects-can-i-get-access-to-github-copilot-for-free

**Comment by manasesjesus** [comment]
You can get access to GitHub Copilot for free through the Student Developer Pack, in case you are a student.

**Comment by diegoleondev** [comment]
This is because GitHub Copilot is closing its beta stage, to launch the paid version, that is why they are no longer accepting new testers

**Comment by Zoltus** [comment]
> I signed up for github copilot for free, but my friend when he wanted to register it couldn't be free anymore

GitHub Copilot was free for the beta stage and sadly that has ended.
You can get Copilot for free if you are a student or maintainer of popular open source project. More information here:
https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-for-your-schoolwork/apply-for-a-student-developer-pack
